### PR TITLE
Add codetour demo

### PR DESCRIPTION
**Pull request by mattseddon**
@shcheklein this is the prototype/demo I talked about building in the planning meeting last week:

https://user-images.githubusercontent.com/37993418/189579219-44aec742-0786-4080-addd-00bc1f07657d.mov

I think this could be a good entry point into the ecosystem with some work.

### CODE REVIEW

**Comment by mattseddon** [comment]
Wasn't sure about the best place to put the demo so made this PR.

cc @sroy3 @julieg18 @wolmir @maxagin

File: .tours/get-started-with-dvc.tour [hunk 1]
```diff
@@ -0,0 +1,77 @@
+{
```
Comment (mattseddon) [inline .tours/get-started-with-dvc.tour:1]: The main goal here was to make a low-cost prototype not something that is perfect.

**Comment by stephr0y** [comment]
This looks really cool. Especially since it's built-in and it looks really simple to create and add/remove steps.

**Comment by shcheklein** [comment]
@mattseddon looks cool. Usually there is a problem of maintaining this though. Any idea how people will find this? How should we guide them into VS Code + this extension + run the tutorial?

**Comment by mattseddon** [comment]
> @mattseddon looks cool. Usually there is a problem of maintaining this though. 

Agreed we would want to rely on links to explanations. Even those would require maintenance. The cost still might be too high.

> Any idea how people will find this? How should we guide them into VS Code + this extension + run the tutorial?

I would add a new page/section to https://dvc.org/doc/start, we could also add a section to our marketplace listing (readme) detailing that this is one way to get started.

I have also added extension recommendations to this PR and the required extensions into the `devcontainer.json`.

**Comment by shcheklein** [comment]
@mattseddon could you please migrate this to the https://github.com/iterative/example-repos-dev in some form? 

This repo is autogenerated and this branch affects the Studio demo project (it's visible to every new user) - e.g. https://studio.iterative.ai/user/shcheklein/projects/example-get-started-yj0q1nba3g

**Pull request closed without merging**